firestore/integration_test.go

```diff
@@ -155,21 +171,17 @@ func createIndexes(ctx context.Context, dbPath string) {
 				Fields:     adminPbIndexFields,
 			},
 		}
-		go func(req *adminpb.CreateIndexRequest, i int) {
-			op, createErr := iAdminClient.CreateIndex(ctx, req)
-			if createErr != nil {
-				log.Fatalf("CreateIndex: %v", createErr)
-			}
+		op, createErr := iAdminClient.CreateIndex(ctx, req)
```

Comment:
also for my understanding, can you explain why this is changing?

Reply:
Earlier indexes used to get created in parallel. But I have serialized the creation.
When integration tests are run on a newly created database, they fail with error if indexes are created in parallel:

```
cloud_datastore.metadata/FirestoreMetadataWrite.BootstrapDatabase;Concurrent access
```

i.e. there are other bootstrap activities at first write time.